skimage/restoration/deconvolution.py

```diff
@@ -365,13 +365,22 @@ def richardson_lucy(image, psf, iterations=50, clip=True):
     ----------
     .. [1] http://en.wikipedia.org/wiki/Richardson%E2%80%93Lucy_deconvolution
     """
+    direct_time = lambda n, m, k, l: k*l * n*m
+    def fft_time(m, n, k, l):
+        return m*np.log(m) + n*np.log(n) + k*np.log(k) + l*np.log(l)
+
+    time_ratio = 71.468 * fft_time(*image.shape, *psf.shape)
```

Comment:
@emmanuelle You don't need to recast as tuple, and, even more, you don't need to cast as list!

```python
>>> (5, 9) + (5, 8)
(5, 9, 5, 8)
```
So `fft_time(*(image.shape + psf.shape))` should work.